cypress/integration/form-validation/in-list.spec.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+
+
+describe('Form validation', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:4200/forms-and-validation/in-a-list');
+    });
+
+    it('should display page title', () => {
+        cy.get('[data-cy=pxb-toolbar]').should('contain', 'In a List');
+    });
+
+    it('should display ip address left aligned', () => {
+        cy.get('[data-cy=ip-address]').invoke('css', 'text-align').should('equal', 'left')
+    });
+
+    it('should display disabled toggle in #424e54 opacity 0.3', () => {
```

Comment:
There's no 'disabled' switch as part of this example; should this be "should display unselected toggle" ?

Reply:
changed this to unselected and the other as selected